src/kernel/mconfig_dot.ml

```diff
@@ -320,15 +338,15 @@ let find_project_context start_dir =
     try
       Some (
         List.find_map [
-            ".merlin"; "dune-project"; "dune-workspace"
+          ".merlin.custom"; ".merlin"; "dune-project"; "dune-workspace"
```

Comment:
I think we should move this to the end of the list because it is the least-expected case.

```suggestion
          ".merlin"; "dune-project"; "dune-workspace"; ".merlin.custom"
```

Reply:
But we want it to be picked up first: if you move it at the end, it will pick up .merlin first so we can't override the logic (or I am mistaken ?)

Reply:
You are right, let's keep this order.